R/ggnet.R

```diff
@@ -209,7 +209,7 @@ ggnet <- function(
 
   # -- packages ----------------------------------------------------------------
 
-  require_namespaces(c("network", "sna", "scales"))
+  rlnag::check_installed(c("network", "sna", "scales"))
```

Comment:
typo

Reply:
Fixed in [`4e13804` (#531)](https://github.com/ggobi/ggally/pull/531/commits/4e13804848c689a3b8d305553eeee567c0729214#diff-00cd7014ee3b002ff895408c2af1b54a5956acac87c581b3818a5e6b3e2fa66aR212)